editions/tw5.com/tiddlers/wikitext/Widgets in WikiText.tid

```diff
@@ -1,11 +1,11 @@
 caption: Widgets
 created: 20131205160840915
-modified: 20220513113347606
+modified: 20220728190445097
 tags: WikiText
 title: Widgets in WikiText
 type: text/vnd.tiddlywiki
 
-Widgets provide rich functionality within WikiText. They have the same syntax as [[HTML elements|HTML in WikiText]], but the tag name always starts with `$`. For example:
+Widgets provide rich functionality within WikiText. They have the same syntax as HTML elements, but the tag name always starts with `$`. For example:
```

Comment:
Why did you remove the link to "HTML in WikiText"? All the information in that tiddler is relevant for widgets.